src/mon/OSDMonitor.cc

```diff
@@ -3497,6 +3497,7 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
   }
   else if (prefix == "osd dump" ||
 	   prefix == "osd tree" ||
+	   prefix == "osd tree-ls" ||
 	   prefix == "osd ls" ||
```

Comment:
Seems by accident you changed the mode of this file to be `executable `...

Reply:
thanks, i change it to default